plugins/plugin-sass/README.md

```diff
@@ -30,6 +30,13 @@ module.exports = {
 
 ## Plugin Options
 
-| Name     |   Type    | Description                                                                                                                                                                                                                                                                            |
-| :------- | :-------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `native` | `boolean` | If true, the plugin will ignore the npm version of sass installed locally for the native Sass CLI [installed separately](https://sass-lang.com/install). This involves extra set up, but the result can be [up to 9x faster.](https://stackoverflow.com/a/56422541) Defaults to false. |
+| Name             |                   Type                   | Description                                                                                                                                                                                                                                                                            |
+| :--------------- | :--------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `native`         |                `boolean`                 | If true, the plugin will ignore the npm version of sass installed locally for the native Sass CLI [installed separately](https://sass-lang.com/install). This involves extra set up, but the result can be [up to 9x faster.](https://stackoverflow.com/a/56422541) Defaults to false. |
```

Comment:
@FredKSchott I just noticed this comment. I don’t think this is accurate?

The “Dart Sass (JS)” is referring to actually using [`require('sass')` in a Node project](https://sass-lang.com/documentation/js-api#usage) and using the Node wrapper. We’re using the CLI executable, which means we already are using the 9x faster version. Unless I’m mistaken?

Reply:
Ah nevermind; read the [blog post](https://itnext.io/the-css-preprocessor-dilemma-node-sass-or-dart-sass-32a0a096572) attached to that SO ticket. Wow! This is accurate! TIL

Reply:
Right? Pretty wild